vm/devices/get/guest_emulation_device/src/lib.rs

```diff
@@ -234,6 +247,9 @@ pub struct GuestEmulationDevice {
     save_restore_buf: Option<Vec<u8>>,
     last_save_restore_buf_len: usize,
 
+    #[inspect(skip)]
```

Comment:
should this really be skipped in inspect, or can we just derive inspect on the enum above?